solr/core/src/test/org/apache/solr/handler/TestStressThreadBackup.java

```diff
@@ -334,7 +335,7 @@ private void validateBackup(final Path backup) throws IOException {
     final int numRealDocsExpected = Integer.parseInt(m.group());
 
     try (Directory dir = FSDirectory.open(backup)) {
-      TestUtil.checkIndex(dir, 0, true, true, null);
+      TestUtil.checkIndex(dir, CheckIndex.Level.MIN_LEVEL_FOR_SLOW_CHECKS, true, true, null);
```

Comment:
why this choice?  I would expect that merely checksum (level 1) is fine for backups

Reply:
Keep it actually; you make a good point that this is a stress test.